### PR TITLE
Xyverz bastyl

### PR DESCRIPTION
**Pull request by Xyverz**
Adding my Bastyl keymap.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/keymaps/xyverz/config.h [hunk 1]
```diff
@@ -48,9 +48,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     #define RGBLIGHT_VAL_STEP 17
 #endif
 
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/crkbd/keymaps/xyverz/config.h:51]: Whoops

File: keyboards/crkbd/keymaps/xyverz/rules.mk [hunk 2]
```diff
@@ -1,2 +1,6 @@
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/crkbd/keymaps/xyverz/rules.mk:1]: Whoops

File: keyboards/crkbd/keymaps/xyverz/config.h (hunk 1 above)
Comment (Xyverz) [reply keyboards/crkbd/keymaps/xyverz/config.h]: LOL, thanks! Fixed. =D

File: keyboards/crkbd/keymaps/xyverz/rules.mk (hunk 2 above)
Comment (Xyverz) [reply keyboards/crkbd/keymaps/xyverz/rules.mk]: LOL, thanks! Fixed. =D

**Review by drashna — APPROVED** [review]

File: keyboards/hidtech/bastyl/keymaps/xyverz/keymap.c [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                             _______, KC_DEL,  _______,     _______, _______, _______, \
+                                      MACLOCK, _______,     _______, WINLOCK
+  ),
+
+[_ADJUST] = LAYOUT(
+  _______, _______, _______, _______, _______, _______,     _______, _______, _______, _______, _______, _______, \
+  _______, RESET,   _______, _______, _______, _______,     _______, _______, _______, _______, _______, _______, \
+  _______, _______, _______, _______, _______, _______,     _______, WINDOWS, MAC,     _______, _______, _______, \
+  _______, _______, _______, _______, _______, _______,     _______, _______, _______, _______, _______, _______, \
+                             _______, _______, _______,     _______, _______, _______, \
```
Comment (fauxpark) [inline keyboards/hidtech/bastyl/keymaps/xyverz/keymap.c:80]:
```suggestion
  KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,        KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSLS,
  KC_TAB,  KC_QUOT, KC_COMM, KC_DOT,  KC_P,    KC_Y,        KC_F,    KC_G,    KC_C,    KC_R,    KC_L,    KC_SLSH,
  KC_GRV,  KC_A,    KC_O,    KC_E,    KC_U,    KC_I,        KC_D,    KC_H,    KC_T,    KC_N,    KC_S,    KC_MINS,
  KC_LSFT, KC_SCLN, KC_Q,    KC_J,    KC_K,    KC_X,        KC_B,    KC_M,    KC_W,    KC_V,    KC_Z,    KC_RSFT,
                             LOWER,   KC_BSPC, KC_DEL,      KC_ENT,  KC_SPC,  RAISE,
                                      KC_LCTL, KC_LALT,     KC_RGUI, KC_RCTL
  ),

[_MAC] = LAYOUT(
  KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,        KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSLS,
  KC_TAB,  KC_QUOT, KC_COMM, KC_DOT,  KC_P,    KC_Y,        KC_F,    KC_G,    KC_C,    KC_R,    KC_L,    KC_SLSH,
  KC_GRV,  KC_A,    KC_O,    KC_E,    KC_U,    KC_I,        KC_D,    KC_H,    KC_T,    KC_N,    KC_S,    KC_MINS,
  KC_LSFT, KC_SCLN, KC_Q,    KC_J,    KC_K,    KC_X,        KC_B,    KC_M,    KC_W,    KC_V,    KC_Z,    KC_RSFT,
                             LOWER,   KC_BSPC, DELGUI,      KC_ENT,  KC_SPC,  RAISE,
                                      KC_LCTL, KC_LALT,     KC_RGUI, KC_RCTL
  ),

[_LOWER] = LAYOUT(
  KC_F11,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,       KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F12,
  RESET,   _______, _______, KC_UP,   _______, _______,     _______, KC_PSCR, KC_SLCK, KC_PAUS, _______, _______,
  KC_CAPS, _______, KC_LEFT, KC_DOWN, KC_RGHT, KC_HOME,     KC_PGUP, _______, KC_PLUS, KC_LCBR, KC_RCBR, _______,
  _______, _______, KC_MPRV, KC_MPLY, KC_MNXT, KC_END,      KC_PGDN, KC_MUTE, KC_VOLD, KC_VOLU, _______, _______,
                             _______, KC_DEL,  _______,     _______, _______, _______,
                                      MACLOCK, _______,     _______, WINLOCK
  ),

[_RAISE] = LAYOUT(
  KC_F11,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,       KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F12,
  _______, _______, _______, KC_UP,   _______, _______,     _______, KC_PSCR, KC_SLCK, KC_PAUS, _______, RESET,
  KC_CAPS, _______, KC_LEFT, KC_DOWN, KC_RGHT, KC_HOME,     KC_PGUP, _______, KC_EQL,  KC_LBRC, KC_RBRC, _______,
  _______, _______, KC_MPRV, KC_MPLY, KC_MNXT, KC_END,      KC_PGDN, KC_MUTE, KC_VOLD, KC_VOLU, _______, _______,
                             _______, KC_DEL,  _______,     _______, _______, _______,
                                      MACLOCK, _______,     _______, WINLOCK
  ),

[_ADJUST] = LAYOUT(
  _______, _______, _______, _______, _______, _______,     _______, _______, _______, _______, _______, _______,
  _______, RESET,   _______, _______, _______, _______,     _______, _______, _______, _______, _______, _______,
  _______, _______, _______, _______, _______, _______,     _______, WINDOWS, MAC,     _______, _______, _______,
  _______, _______, _______, _______, _______, _______,     _______, _______, _______, _______, _______, _______,
                             _______, _______, _______,     _______, _______, _______,
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/hidtech/bastyl/keymaps/xyverz/keymap.c (hunk 3 above)
Comment (Xyverz) [reply keyboards/hidtech/bastyl/keymaps/xyverz/keymap.c]: Fixed.

**Review by fauxpark — APPROVED** [review]

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**